assets/training/model_management/src/azureml/model/mgmt/processors/common/vision_utils.py

```diff
@@ -58,8 +60,13 @@ def process_image(img: pd.Series) -> pd.Series:
         return img
     elif isinstance(image, str):
         if _is_valid_url(image):
-            image = requests.get(image).content
-            return pd.Series(image)
+            try:
+                response = requests.get(image)
+                response.raise_for_status()  # Raise exception in case of unsuccessful response code.
+                image = response.content
+                return pd.Series(image)
+            except requests.exceptions.RequestException as ex:
```

Comment:
Can we log the url in the exception so that the user knows which records to correct?

Reply:
After testing it seems the url is logged as part of the exception. Resolving